asset-transfer-basic/rest-api-typescript/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "asset-transfer-basic",
```

Comment:
1. I suggest you to specify node, and npm version requirements on this package file.
2. I see that npm version is quite low ( `lockfileVersion` is 1 )

Reply:
Hi @kingworm, thanks, I'll add that